docs/source/nlp/quantization.rst

```diff
@@ -133,15 +132,60 @@ Known issues
 * The supported and tested model family is Llama2. Quantizing other model types is experimental and may not be fully supported.
 
 
-Please refer to the following papers for more details on quantization techniques.
+Quantization-Aware Training (QAT)
+---------------------------------
 
-References
-----------
+QAT is the technique of fine-tuning a quantized model to recover model quality degradation due to quantization.
+While QAT requires much more compute resources than PTQ, it is highly effective in recovering model quality.
+To perform QAT on a calibrated model from PTQ, you need to further fine-tune the model on a downstream task using a small dataset before exporting to TensorRT-LLM.
+You can reuse your training pipeline for QAT.
+Since QAT is done after PTQ, the supported model families are the same as for PTQ.
+
+
+Example
+^^^^^^^
+
+The example below shows how to perform PTQ and QAT on a Supervised Finetuned Llama2 7B model to INT4 precision using tensor parallelism of 8 on 8x A5000 24GB GPUs.
+For bigger models like Llama2 70B, you may need to use one or more DGX H100 nodes with 8x 80GB GPUs each.
+
+The example is a modified version of the `SFT with Llama 2 playbook <https://docs.nvidia.com/nemo-framework/user-guide/latest/playbooks/llama2sft.html>`_.
+Please refer to the playbook for more details on setting up a BF16 NeMo model and the ``databricks-dolly-15k`` instruction dataset.
+
+First we will run the SFT example command from the playbook as-is to train a Llama2 7B SFT model for 100 steps.
+Make sure to change ``trainer.max_steps=50`` to ``trainer.max_steps=100`` for the ``examples/nlp/language_modeling/tuning/megatron_gpt_finetuning.py`` script.
+This will produce a model checkpoint with validation loss approximately ``1.15`` that we will use for PTQ and QAT next.
+
+For Quantization, we use a modified version of the sft script and config file which includes the quantization and TRT-LLM export support.
+Along with the new parameters, make sure to pass the same parameters you passed for SFT training except the model restore path will be the SFT output ``.nemo`` file.
```

Comment:
WE recommend using a small learning rate for QAT. See here - https://github.com/NVIDIA/TensorRT-Model-Optimizer/tree/main/llm_qat#qat-example-workflow

Reply:
Tried 1e-6 instead of 5e-6 SFT learning rate but the model trains very very slowly, 100 steps only improves loss from 1.31 to 1.30 so I guess we need to use the same LR as the SFT LR likely because SFT LR is already very small

Reply:
The recommendation we give is to use a small learning rate such as "QAT for 1-10% original training duration and a small learning rate, e.g. 1e-5 for Adam optimizer"

In cases such as these SFT LR is already small - QAT happened to work with the original LR. But this does not need to be the case always.

Lets make this clear.  Instead of saying the training parameters are kept the same, can we say we adjust the QAT LR to be a small value.

Reply:
Updated as follows:

```
As a rule of thumb, we recommend QAT for 1-10% original training duration and a small learning rate, e.g. 1e-5 for Adam optimizer.
If you are doing QAT on an SFT model where learning rates and finetuning dataset size are already small, you can continue using the same SFT learning rate and dataset size as a starting point for QAT.
```